### DISCUSSION TITLE
Changelog: Phantom 23.17.0

### DISCUSSION THREAD

**Discussion by adamdelphantom** [phantom/docs#148] [Announcements]
### **New Features & Improvements 🎉**

- 🔀⛓️ **Cross-Chain Swaps** - You can now easily and quickly bridge tokens across Solana, Ethereum, and Polygon directly in Phantom!
    
![crosschainswaps](https://github.com/phantom/docs/assets/64601408/4a42c8ff-3449-49f1-afa5-d314c8e1796f)

   
- 🗺️ **Push Notifications for Quests**

****Bug Fixes 🐛****

- 🥩 **Marinade Staking** - Fixed a bug where users were unable to unstake from Marinade.
- 🚫 **Error Rendering** - Fixed a bug where users were seeing raw errors.

**Comment by shtefcs** [comment]
Cool update. I think creating good UX for mobile devices is crucial, deep links etc https://github.com/orgs/phantom/discussions/141
